ui/components/editblock/base/_index.scss

```diff
@@ -303,3 +303,123 @@
         height: 3rem;
     }
 }
+
+
+/**
+ * Inline Edit Block
+ */
+
+
+.nds-editblock_inline {
+    .nds-edit-block-edit__container {
+        border: 1px solid #dddbda;
```

Comment:
@apurv3039 Could we please use colors from the design-tokens? @avnesh-neev Could you please help Apurv with this.